### PR TITLE
Draft PR - adding hero styles

### PR DESCRIPTION
**Pull request by arjun87743**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://44-herostyles--world-bank--aemsites.hlx.live/

### CODE REVIEW

File: blocks/hero/hero.css [hunk 1]
```diff
@@ -32,3 +32,91 @@ main .hero-container {
   width: 100%;
   height: 100%;
 }
+
+.hero.block {
```
Comment (thomsebastin) [inline blocks/hero/hero.css:36]: You can use nesting to namespace this better. For example:

```css
.hero {
    &.block {
        ... code
    }
}
```
Something like this.

File: blocks/hero/hero.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+export default function decorate(block) {
+  const pictureElement = block.querySelector('div picture');
+  const titleElement = block.querySelector('div:nth-child(2) > div > p');
+  const descriptionElement = block.querySelector('div:nth-child(3) > div > p');
+  const buttonElement = block.querySelector('div:nth-child(4) > div > p');
+  const linkElement = block.querySelector('div a');
+
+  // Create the hero container
+  const heroContainer = document.createElement('div');
```
Comment (thomsebastin) [inline blocks/hero/hero.js:9]: lets change to use `dom-utils` added by @rrusher. Should make it much cleaner.

**Review by thomsebastin — CHANGES_REQUESTED** [review]

**Comment by arjun87743** [comment]
@thomsebastin The scope of this ticket has changed and @skumawat1 will be working on this.

**Comment by thomsebastin** [comment]
@arjun87743  kindly close the PR if not needed.

**Pull request closed without merging**